src/Servers/IIS/src/AspNetCoreModuleV2/RequestHandlerLib/environmentvariablehelpers.h

```diff
@@ -240,6 +242,26 @@ class ENVIRONMENT_VAR_HELPERS
             goto Finished;
         }
 
+        if (pHttpsPort != nullptr)
+        {
+            pEnvironmentVarTable->FindKey((PWSTR)ASPNETCORE_HTTPS_PORT_ENV_STR, &pIISHttpsPort);
+            if (pIISHttpsPort != NULL)
+            {
+                // user defined ASPNETCORE_HTTPS_PORT in configuration, don't override it
+                pIISHttpsPort->Dereference();
+            }
+            else
+            {
+                pIISHttpsPort = new ENVIRONMENT_VAR_ENTRY();
```

Comment:
ew naked new.

Reply:
`environmentvariablehelpers.h` is full of that